frontend/src/audio.html

```diff
@@ -0,0 +1,84 @@
+---
+pagination:
+  data: audios
+  size: 1
+  alias: audio
+permalink: "/audios/{{ audio.objectID | slug }}/"
+layout: base
+templateEngineOverride: md, njk, html
+eleventyComputed:
+  title: "{{ audio.objectID }}"
```

Comment:
I'm not sure we need to set title for this page. I mean, on mobiles we don't want it to be shown in the top in green.

Reply:
we dont need to set a new variable for audio Id, its available in JSON as objectID, so removed.